common/scripts/changelog/collect.mjs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
```

Comment:
Can you add comments to the top of each script file with a quick 'what this does/is for' summary

Reply:
Would be worth adding a `docs/` doc also with how the tooling handles all this new stable and beta change files